contracts/modules/Affiliates.sol

```diff
@@ -191,6 +192,7 @@ contract Affiliates is State, AffiliatesEvents {
 		} else {
 			emit PayTradingFeeToAffiliateFail(
 				referrer,
+				tx.origin, // trader
```

Comment:
add trader as a parameter to he function
see detales in the commend above #L184

Reply:
Changes has been made